### PR TITLE
Weapon name & static constant readonly variables

### PR DESCRIPTION
**Pull request by Fueredoriku**
- Added weapon names!
- Made a way of accessing all currently "available game assets" like which weapons should show up in the game.
image
image

### CODE REVIEW

**Review by toberge — CHANGES_REQUESTED** [review]
LGTM but I have a couple of nitpicks.

File: Assets/Scripts/Auction/Item.cs [hunk 1]
```diff
@@ -11,8 +11,14 @@ public enum AugmentType
 public class Item : ScriptableObject
 {
     public string displayName;
+    public string secretName;
     public string displayDescription;
```
[the remark this answers is not in the document]
Comment (toberge) [reply Assets/Scripts/Auction/Item.cs:15]: Too much of a hassle though, sooooooooo nah

**Review by toberge — APPROVED** [review]
Let's get this boi in

File: Assets/Scripts/Auction/Item.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Fueredoriku) [reply Assets/Scripts/Auction/Item.cs:15]: I will create a separate issue for this. 👍

**Pull request merged**